rclcpp_components/src/component_manager.cpp

```diff
@@ -22,6 +22,7 @@
 
 #include "ament_index_cpp/get_resource.hpp"
 #include "class_loader/class_loader.hpp"
+#include "rclcpp/rclcpp.hpp"
```

Comment:
Recommend using more specific includes. What is used that wasn't brought in by `component_manager.hpp`?

Reply:
I can drop this one, I can't remember what it was. I don't think something wasn't compiling, it was just more of a "include what you use" kind of thing.